plugins/sql_extract/src/sql_extract/component.py

```
@@ -0,0 +1,264 @@
+                df = pl.DataFrame(data)
+
+                logger.debug(
+                    "Запрос выполнен: %s строк, %s колонок",
+                    len(df),
+                    len(df.columns),
+                )
+
+                return df
+
+        except Exception as e:
+            logger.error(f"Ошибка выполнения запроса: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)